lib/kitchen/directions/bake_figure/caption_container.xhtml.erb

```diff
@@ -0,0 +1,13 @@
+<div class="os-caption-container">
+    <span class="os-title-label"><%= @title_label %></span>
+    <span class="os-number"><%= @number %></span>
+    <%- if @title %>
+      <span class='os-divider'> </span>
+      <span class='os-title' data-type='title' id=<%= @title.id %>><%= @title.children %></span>
+    <% end %>
+    <%- if @caption %>
+      <span class='os-divider'> </span>
+      <span class='os-caption'><%= @caption.children %></span>
```

Comment:
This irb template fixed the whitespace problem inside `os-caption` span children, but it caused to failed the recipe spec for organic chemistry because of removing the additional `m:` math namespaces:  
![image](https://user-images.githubusercontent.com/55203598/183448093-740aaf7e-a032-464f-b802-24b9ddd1922c.png)